gapic-generator-cloud/expected_output/speech/lib/google/cloud/speech/v1/speech_client.rb

```diff
@@ -43,7 +40,7 @@ class SpeechClient
           DEFAULT_SERVICE_PORT = 443
 
           # The default set of gRPC interceptors.
-          GRPC_INTERCEPTORS = []
+          GRPC_INTERCEPTORS = [].freeze
```

Comment:
I don't understand why this is being frozen. The whole point of this constant is to be a place to define the interceptors for the clients.

Reply:
Fixed.